librz/core/cmd/cmd_flag.c

```diff
@@ -686,15 +629,14 @@ static bool adjust_offset(RzFlagItem *flag, void *user) {
 	return true;
 }
 
-static void print_space_stack(RzFlag *f, int ordinal, const char *name, bool selected, PJ *pj, int mode) {
-	bool first = ordinal == 0;
-	switch (mode) {
-	case 'j': {
+static void print_space_stack(RzFlag *f, int ordinal, const char *name, bool selected, RzCmdStateOutput *state) {
+	PJ *pj = state->d.pj;
```

Comment:
not sure about this. What happens on some architectures when actually the output mode is another one? This may be an invalid reference. Is it alright?

Reply:
I usually just directly use `state->d.pj` and that's it.